NotificationHubs/src/com/windowsazure/messaging/CollectionResult.java

```diff
@@ -14,29 +14,41 @@
     private String continuationToken;
     private final List<Registration> registrations = new LinkedList<>();
```

Comment:
`LinkedList` in the wild, don't see that often 😆

Reply:
Yeah, this code base is full of gems like that!